description.go

```diff
@@ -1,21 +1,45 @@
 package main
 
 import (
-	"bytes"
 	"context"
+	_ "embed"
 	"fmt"
-	"os/exec"
+	"regexp"
 	"strings"
 
-	"github.com/russross/blackfriday"
+	"github.com/charmbracelet/glamour"
 )
 
+//go:embed description.json
+var descriptionJSONBytes []byte
+
+// reformatForControl reformats the wrapped description
+// to conform to Debian’s control format.
 func reformatForControl(raw string) string {
-	// Reformat the wrapped description to conform to Debian’s control format.
-	for strings.Contains(raw, "\n\n") {
-		raw = strings.Replace(raw, "\n\n", "\n.\n", -1)
+	output := ""
+	next_prefix := ""
+	re := regexp.MustCompile(`^ \d+\. `)
+	for _, line := range strings.Split(strings.TrimSpace(raw), "\n") {
+		// Remove paddings that Glamour currently add to the end of each line
+		line = strings.TrimRight(line, " ")
+
+		prefix := next_prefix
+		if strings.HasPrefix(line, " * ") {
+			prefix = ""
+			next_prefix = "  "
+		}
+		if re.MatchString(line) {
+			prefix = ""
+			next_prefix = "   "
+		}
+		if line == "" {
+			line = "."
+			prefix = ""
+			next_prefix = ""
+		}
+		output += " " + prefix + line + "\n"
 	}
-	return strings.Replace(raw, "\n", "\n ", -1)
+	return output
```

Comment:
The processing looks really nice! But I think adding a little unit test for all the prefix, next_prefix could be really great in helping further regression. What do you think?

Reply:
That's a great idea!  Thank you!  Will be learning how to do it soon  (I'm still pretty new to test-driven development :sweat_smile:)